Can't restart Codewind node project in debug mode

Using a newly created Codewind node project (not Appsody) when I try to restart it in debug mode I get an error dialog popping up with the message: projectStatusController.appStatusContainerStopped.  This is coming from a projectStatusChanged event:

```
[10/01/20 19:35:05 node1] [INFO] Emitting event 
 message: projectStatusChanged
 data: {
  "projectID": "c6165f60-33df-11ea-a2a1-998725219ae8",
  "appStatus": "stopped",
  "appErrorStatus": "projectStatusController.appStatusContainerStopped",
  "detailedAppStatus": {
    "severity": "ERROR",
    "message": "projectStatusController.appStatusContainerStopped",
    "notify": true
  }
}
```

I am using the latest 0.8.0 Codewind and Eclipse IDE.  I tried both on Windows and MAC with the same results.

Full log file: 
[pfelog.txt](https://github.com/eclipse/codewind/files/4047618/pfelog.txt)

I see it too

```
[10/01/20 19:34:54 nodeproj] [ERROR] runScript : 1105 | + set +x

[10/01/20 19:34:54 nodeproj] [ERROR] runScript : 1105 | + nodemon --exec npm run debug

[10/01/20 19:34:54 nodeproj] [INFO] Project start for restart was successful
[10/01/20 19:34:54 User.js] [INFO] Updating status for project fe9913f0-33df-11ea-8330-65459ad4e4f2.
[10/01/20 19:34:54 nodeproj] [INFO] Application state changed for project: fe9913f0-33df-11ea-8330-65459ad4e4f2 from: stopped, to: starting
[10/01/20 19:34:54 nodeproj] [INFO] Emitting event
 message: projectStatusChanged
 data: {
  "projectID": "fe9913f0-33df-11ea-8330-65459ad4e4f2",
  "appStatus": "starting"
}
[10/01/20 19:34:54 nodeproj] [INFO] Internal port: 3000
[10/01/20 19:34:54 nodeproj] [INFO] Internal debug port: 9229
[10/01/20 19:34:54 nodeproj] [INFO] Exposed port: 32816
[10/01/20 19:34:54 nodeproj] [INFO] Exposed debug port: 32815
[10/01/20 19:34:54 nodeproj] [INFO] All the exposed container ports: ["3000","9229"]
[10/01/20 19:34:54 nodeproj] [INFO] All the host ports: ["32816","32815"]
[10/01/20 19:34:54 nodeproj] [INFO] Emitting event
 message: projectRestartResult
 data: {
  "operationId": "f6a93019625b3e7a3c2e4e86c9946816",
  "projectID": "fe9913f0-33df-11ea-8330-65459ad4e4f2",
  "status": "success",
  "startMode": "debugNoInit",
  "ports": {
    "exposedPort": "32816",
    "internalPort": "3000",
    "exposedDebugPort": "32815",
    "internalDebugPort": "9229"
  },
  "containerId": "9dc9572f8e9c1ea1f171b87d32c577ec4e3f94f63d75eced46d12b70ddceb3f0"
}
[10/01/20 19:35:06 nodeproj] [ERROR] getContainerStatus : 167 | | The command docker exec,cw-nodeproj-fe9913f0-33df-11ea-8330-65459ad4e4f2,/scripts/noderun.sh,isActive failed with exit code 1
[10/01/20 19:35:06 nodeproj] [INFO] pingInTransitApplications: Application state for project fe9913f0-33df-11ea-8330-65459ad4e4f2 has changed from starting to stopped
[10/01/20 19:35:06 nodeproj] [INFO] Emitting event
 message: projectStatusChanged
 data: {
  "projectID": "fe9913f0-33df-11ea-8330-65459ad4e4f2",
  "appStatus": "stopped",
  "appErrorStatus": "projectStatusController.appStatusContainerStopped",
  "detailedAppStatus": {
    "severity": "ERROR",
    "message": "projectStatusController.appStatusContainerStopped",
    "notify": true
  }
}
```

The message is cleaned up now but it still won't start in debug mode.

![image](https://user-images.githubusercontent.com/9484320/72261095-323d9880-35e2-11ea-98b6-b4f1d1b76172.png)

The app.log has the following at the end:

+ nodemon@2.0.2
updated 1 package in 3.999s
[33m[nodemon] 2.0.2[39m
[33m[nodemon] to restart at any time, enter `rs`[39m
[33m[nodemon] watching dir(s): *.*[39m
[33m[nodemon] watching extensions: js,mjs,json[39m
[32m[nodemon] starting `npm run debug`[39m
npm ERR! missing script: debug

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-01-10T08_51_54_664Z-debug.log
[31m[nodemon] app crashed - waiting for file changes before starting...[39m

--- a/package.json
+++ b/package.json
@@ -10,6 +10,11 @@
   },
   "scripts": {
     "start": "node $npm_package_config_entrypoint"
+    "debug": "node --inspect=0.0.0.0:9229 $npm_package_config_entrypoint",	
+    "debug:brk": "node --inspect-brk=0.0.0.0:9229 $npm_package_config_entrypoint",	
+    "debug:legacy": "node --debug=0.0.0.0:5858 $npm_package_config_entrypoint",	
+    "test": "nyc mocha --exit",	
+    "dev": "nodemon $npm_package_config_entrypoint"
   },
   "dependencies": {
     "appmetrics-dash": "^5.2.0",

--- a/package.json
+++ b/package.json
@@ -12,9 +12,7 @@
     "start": "node $npm_package_config_entrypoint",
     "debug": "node --inspect=0.0.0.0:9229 $npm_package_config_entrypoint",	
     "debug:brk": "node --inspect-brk=0.0.0.0:9229 $npm_package_config_entrypoint",	
-    "debug:legacy": "node --debug=0.0.0.0:5858 $npm_package_config_entrypoint",	
-    "test": "nyc mocha --exit",	
-    "dev": "nodemon $npm_package_config_entrypoint"
+    "debug:legacy": "node --debug=0.0.0.0:5858 $npm_package_config_entrypoint"
   },
   "dependencies": {
     "appmetrics-dash": "^5.2.0",

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     "entrypoint": "server/server.js"
   },
   "scripts": {
-    "start": "node $npm_package_config_entrypoint"
+    "start": "node $npm_package_config_entrypoint",
     "debug": "node --inspect=0.0.0.0:9229 $npm_package_config_entrypoint",	
     "debug:brk": "node --inspect-brk=0.0.0.0:9229 $npm_package_config_entrypoint",	
     "debug:legacy": "node --debug=0.0.0.0:5858 $npm_package_config_entrypoint",	

